templates/core/index.html

```diff
@@ -706,32 +717,6 @@ <h4>License compliance</h4>
     <hr class="p-separator" />
   </div>
 
-  <div class="row u-vertically-center">
-    <div class="col-6">
-      <h4>Software ecosystem, ready to go</h4>
-      <p class='p-heading--five'>Thousands of applications built to work across devices</p>
-      <p>A standard platform helps publishers support multiple devices without recompiling. Everybody wants to support Ubuntu - it’s the most widely deployed Linux in the world.</p>
-      <p><a class="p-button--neutral is-inline" href="https://snapcraft.io/store" style="margin-left: 0;"><span class="p-link--external">Visit the Snap Store</span></a></p>
-    </div>
-    <div class="col-6 u-align--center u-hide--small">
-      {{
-      image(
-      url="https://assets.ubuntu.com/v1/26e55529-snapstore.png",
-      alt="",
-      height="255",
-      width="477",
-      hi_def=True,
-      loading="lazy",
-      attrs={"class": "p-image--bordered"}
-      ) | safe
-      }}
-    </div>
-  </div>
-
-  <div class="u-fixed-width">
-    <hr class="p-separator" />
-  </div>
-
   <div class="row u-vertically-center">
     <div class="col-6 u-hide--small u-align--center">
```

Comment:
Think it would be better to switch the image and text position so texts on the left and image on the right

Reply:
Done